deploy/helm/README.md

```diff
@@ -0,0 +1,87 @@
+# OpenEBS Node Disk Manager Helm Chart
+
+[OpenEBS Node Disk Manager](https://github.com/openebs/node-disk-manager) aims to make it easy to manage the disks 
+attached to the node on a Kubernetes cluster.
+Please visit the [link](https://github.com/openebs/node-disk-manager) to know more about node disk manager.
```

Comment:
This shouldn't be there / redundant.. we are already in this repo.

Reply:
done.